app/src/components/tasks/CompletionModal.tsx

```diff
@@ -76,7 +78,7 @@ export default function CompletionModal(props: CompletionModalProps) {
             <p>Nice work! Way to show up for the Chinatown communities.</p>
             <p>You can use this ticket towards a chosen giveaway prize of your choice.</p>
             {/* TODO: navigate to raffle page */}
-            <PrimaryButton>Enter my raffle ticket</PrimaryButton>
+            <PrimaryButton onClick={() => {navigate('/raffles')}}>Enter my raffle ticket</PrimaryButton>
             <SecondaryButton onClick={() => { setIsActive(false) }}>Enter Later</SecondaryButton>
```

Comment:
this should probably go back to the borough task page that the user was on, like the cancel button

Reply:
What we added seems to go back to home instead of the task list, but we can also fix later if you want to get this PR merged

Reply:
@jacoblb64 So just to be clear, both buttons should go back to the borough task page?